apps/trading/lib/config/env.ts

```diff
@@ -17,5 +17,9 @@ export const ENV = {
     process.env['NX_TRADING_SENTRY_DSN']
   ),
   envName: windowOrDefault('NX_VEGA_ENV', process.env['NX_VEGA_ENV']),
-  vegaUrl: windowOrDefault('NX_VEGA_URL', process.env['NX_VEGA_URL']),
+  vegaUrl: windowOrDefault('NX_VEGA_URL', process.env['X_VEGA_URL']),
+  explorerUrl: windowOrDefault(
```

Comment:
chore (blocking): i think it is confusing to have this duplication in both libs/environment and here. looks like you are using the env hook to get the value, so this here can be deleted imo.

Reply:
This was my mistake when we were talking about it. My bad @MadalinaRaicu